CHANGELOG.md

```diff
@@ -15,7 +15,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
     _Security_ in case of vulnerabilities.
  -->
 
-## Unreleased
+## [0.16.0] - 2023-12-11
```

Comment:
nit: can you add an unreleased section so changes can just be added in the PR right away please? :pray: